projects/Mallard/src/screens/article/slider/SliderTitle.tsx

```diff
@@ -10,6 +10,8 @@ const SLIDER_FRONT_HEIGHT = DeviceInfo.isTablet()
         : 90
     : 60
 
+const FIRST_SUBTITLE_DATE = new Date('2020-02-17').getTime()
```

Comment:
@blishen we should think about what date this is - I'm hope hoping it's fine to hard code it